internal/pgbackrest/config.go

```diff
@@ -668,39 +710,44 @@ until read -r -t 5 -u "${fd}"; do
     pkill -HUP --exact --parent=0 pgbackrest
   then
     exec {fd}>&- && exec {fd}<> <(:||:)
-    stat --dereference --format='Loaded configuration dated %y' "${filename}"
+    stat --dereference --format='Loaded configuration dated %%y' "${filename}"
```

Comment:
❓ Is this supposed to be here? Seems unrelated to the autogrow changes.

Reply:
we moved from using a normal heredoc to using fmt.Sprintf - with that change i had to escape the character